src/service.js

```diff
@@ -1,56 +1,61 @@
 const axios = require('axios');
-const core = require('@actions/core');
+const logger = require('./logger');
 
 const handleChapterError = (chapter) => {
-  const failedChapterData = {
-    contentPath: JSON.parse(chapter.config.data).path,
-    status: chapter.status,
-    message: chapter.statusText,
+  const filePath = JSON.parse(chapter.config.data).path;
+  const errorMessage = chapter.data.message ? chapter.data.message : chapter.statusText;
+
+  const logBody = {
+    path: filePath,
+    step: 'Chapter',
+    statusCode: chapter.status,
+    message: errorMessage,
```

Comment:
Faltou o request id de novo :trollface: 

```suggestion
  const logBody = {
    path: filePath,
    step: 'Chapter',
    statusCode: chapter.status,
    message: errorMessage,
    requestId: chapter.headers['x-request-id'],
```